src/main/java/com/fasterxml/jackson/databind/deser/SettableAnyProperty.java

```diff
@@ -159,6 +186,23 @@ Object readResolve() {
      */
     public String getPropertyName() { return _property.getName(); }
 
+    /**
+     * Accessor for parameterIndex.
+     * @return -1 if not a parameterized setter, otherwise index of parameter
+     *
+     * @since 2.18
+     */
+    public int getParameterIndex() { return -1; }
+
+    /**
+     * Create an instance of value to set any properties.
```

Comment:
"Create an instance of value to pass through Creator parameter" ?

Reply:
Modified via https://github.com/FasterXML/jackson-databind/pull/4558/commits/0e11a15e3e2626011005d961e160333c764dcec5